software/contrib/menu.py

```diff
@@ -1,6 +1,8 @@
 """See menu.md for details."""
 from bootloader import BootloaderMenu
 
+from contrib.bernoulli_gates import BernoulliGates
+from contrib.euclidean_rhythm import EuclideanRhythm
```

Comment:
One last request, please remove the Euclidean Rhythm references from the menu and we're good to go!

Reply:
Removed.